schemas/SupportedConfiguration.json

```diff
@@ -47,6 +47,16 @@
                     "items": {
                         "type": "number"
                     }
+                },
+                "PowerConfigFullLoad": {
+                    "description": [
+                        "Indicates if the number of power supplies specified",
```

Comment:
Something with this description still does not quite explain it.

Reply:
... if we expect to have the full load (4), then set GPIO to true. If we expect to not have the full load (2), then set the GPIO to false.